src/cordova/apple/xcode/ios/Outline/VpnExtension-Info.plist

```diff
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>XPC!</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.7.0</string>
```

Comment:
Don't you need to change Outline-Info too?

Reply:
Same on macOS

Reply:
It was already changed, it just wouldn't compile 😅